### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gky360**
- Update dependency packages
- Add CI

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,22 +4,25 @@
...
     "prettier:fix": "npm run prettier -- --write",
-    "lint": "npx eslint bin/*"
+    "lint": "run-p lint:*",
+    "lint:eslint": "npx eslint --max-warnings 0 bin/*",
+    "lint:prettier": "npx prettier --check bin/*",
+    "format": "npm run prettier -- --write"
+  },
+  "dependencies": {
+    "license-checker-rseidelsohn": "^4.2.7",
+    "spdx-expression-parse": "^3.0.1",
```
Comment (gky360) [inline package.json]: `spdx-expression-parse` is imported but not used in code. Is this package really needed?

https://github.com/pfnet/licensetool/blob/5bf961aea142831bd44034b36894a9a2188b85ea/bin/licensetool#L9

**Review by makky3939 — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (niboshi) [reply package.json]: Ah, it was used during the initial development to parse composite licenses (such as those with `OR` and `AND`), but now I think such a license is an error without manually configuring in `licensetool.json`, so it's no longer used. It should be safe to remove it.

**Review by niboshi — APPROVED** [review]
Thank you very much!

**Review by niboshi — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (gky360) [reply package.json]: I removed `spdx-expression-parse` in 03c2f56.

**Pull request merged**